### PR TITLE
Update communication-channels.rst

### PR DESCRIPTION
**Pull request by 0x00zer0day**
Removed `cherry_picker` from the "Additional Repositories" section under "Following Python's Development" as its been moved to its own repository according to issue #1093

----
:books: Documentation preview :books:: https://cpython-devguide--1094.org.readthedocs.build/

### CODE REVIEW

**Comment by hugovk** [comment]
Thanks for the PR!

Rather than removing cherry_picker entirely, please could you list its own repo?

**Comment by lysnikolaou** [comment]
> And we could include the other tools listed at https://github.com/python/core-workflow#other-core-workflow-tools ?

Yup, exactly what I was gonna say.

**Comment by 0x00zer0day** [comment]
@hugovk @lysnikolaou, From the tools listed in the "other-core-workflow-tools" what do you suggest I add next to `blurb`. And for the `cherry_picker` Would it be okay to list it below the `blurb` section.

**Comment by 0x00zer0day** [comment]
@arhadthedev Thanks for the suggestion!

File: developer-workflow/communication-channels.rst
Comment (lysnikolaou) [inline developer-workflow/communication-channels.rst:255]: Not sure about the capitalization, but this is probably more accurate.
```suggestion
Other core workflow tools are:
```

File: developer-workflow/communication-channels.rst
Comment (hugovk) [inline developer-workflow/communication-channels.rst:257]: This section is about repos, so we should primarily link to the repo, but can also link to the PyPI page.

Comment (hugovk) [reply developer-workflow/communication-channels.rst:257]: You're not being dumb, it's good to ask questions :)

Maybe something like this?

* [cherry_picker](https://github.com/python/cherry-picker) ([PyPI](https://pypi.org/project/cherry_picker/))

Comment (0x00zer0day) [reply developer-workflow/communication-channels.rst:257]: > You're not being dumb, it's good to ask questions :)
> 
> Maybe something like this?
> 
> * [cherry_picker](https://github.com/python/cherry-picker) ([PyPI](https://pypi.org/project/cherry_picker/))

Thank you for your co-operation, will make the changes mentioned above.

**Comment by 0x00zer0day** [comment]
@hugovk I think I have made the necessary changes? It should be ready for merge.

File: developer-workflow/communication-channels.rst
Comment (hugovk) [inline developer-workflow/communication-channels.rst:274]:
```suggestion
.. _cla-bot: https://github.com/ambv/cla-bot
```

**Review by hugovk — COMMENTED** [review]
Thank you!

**Review by ezio-melotti — APPROVED** [review]

**Review by hugovk — APPROVED** [review]

**Comment by CAM-Gerlach** [comment]
This isn't your fault @0x00zer0day , and I was on a GitHub mental health break and not around to catch this in time, but as a "core workflow" reminder to my fellow core team members, can we *please* try to remember to give PRs an actual useful title at least before we merge them? This not only affects browsing on GitHub, in emails/notifications, etc. but also results in an equally not-useful commit message permanently baked into the commit history. Thanks.

**Comment by 0x00zer0day** [comment]
> This isn't your fault @0x00zer0day , and I was on a GitHub mental health break and not around to catch this in time, but as a "core workflow" reminder to my fellow core team members, can we *please* try to remember to give PRs an actual useful title at least before we merge them? This not only affects browsing on GitHub, in emails/notifications, etc. but also results in an equally not-useful commit message permanently baked into the commit history. Thanks.

It's not their mistake, I was aware of the fact that the PR titles matter as it helps in looking back at the PR history. I should have been careful, I'll keep this in mind for the future PR's. Thank you for being so kind and helpful 😊

**Pull request merged**
